02. Remote and Repository/README.md

```diff
@@ -111,6 +111,47 @@ https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a
 
 ## Cloning an existing repository
 
+### 1. HTTPS
+HTTPS is widely used for cloning repositories due to its ease of setup and compatibility across networks. It’s a great option for users who prefer a simple configuration process.
+
+**How it works**
+- Git communicates over the HTTPS protocol
+- You authenticate using a GitHub Personal Access Token (PAT) when pushing
+- No SSH keys required
+
+**Pros**
+- Works in most case
+- Easiest option for beginners
+- No SSH key setup needed
+- Good for read-only access or quick clones
+
+**Cons**
+- Push operations require entering a Personal Access Token (PAT) unless cached
+- Managing multiple accounts is harder than with SSH
+- Less convenient for frequent use
+- Tokens must be stored securely
+
+### 2. SSH
+SSH is a secure protocol used for authenticating and encrypting communication between your machine and the server. It offers more security and convenience for frequent Git users.
+
+**How it works**
+- Git uses SSH (Secure Shell) to authenticate
+- Authentication is done with your SSH keypair (private + public key)
```

Comment:
explain briefly about public and private key
refer to the next section, setup SSH key pair